lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
should remove AI-Agent.xml at the same time as it's no longer used:
https://github.com/hmcts/ccd-data-store-api/blob/master/lib/AI-Agent.xml